src/services/dashboards/dashboard-customize/modules/VariableMoreButtonDropdown.vue

```diff
@@ -79,26 +85,26 @@ const state = reactive({
     selectedForUpdate: [] as MenuItem[],
 });
 
-const {
-    targetRef,
-    contextMenuRef,
-    variableList,
-    selected,
-} = toRefs(state);
-
 const {
     visibleMenu,
-    hideContextMenu,
+    refinedMenu,
+    contextMenuStyle,
     showContextMenu,
-    fixedMenuStyle,
-    reorderedMenu,
+    hideContextMenu,
+    focusOnContextMenu,
+    initiateMenu,
+    reloadMenu,
+    showMoreMenu,
 } = useContextMenuController({
-    targetRef,
-    contextMenuRef,
-    useReorderBySelection: true,
     useFixedStyle: true,
-    originMenu: variableList,
-    selected,
+    targetRef: toRef(state, 'targetRef'),
+    contextMenuRef: toRef(state, 'contextMenuRef'),
+    useMenuFiltering: true,
+    useReorderBySelection: true,
+    searchText: toRef(state, 'searchText'),
+    selected: toRef(state, 'selected'),
+    menu: toRef(state, 'variableList'),
+    pageSize: 5,
```

Comment:
question: pageSize 5 is too small, I think 😅... Is this the confirmation from PM?